File: Kyu7/vowel_count.js
// Simplier version:
function getCount(str) {
  return (str.match(/[aeiou]/gi) || []).length;
}

// Initial version:
function getCount(str) {
  const vowels = str.match(/[aeiou]/gi);
  const count = vowels === null ? 0 : vowels.length;
  return count;
}

File: Kyu7/vowel_count.py
# def get_count(sentence):
#     count = 0
#     for char in sentence:
#         if char in "aeiou": 
#             count += 1
#     return count

def get_count(sentence):
    return sum(char in "aeiou" for char in sentence)

print(get_count("aeiou")) # 5
print(get_count("bcdfghjklmnpqrstvwxz y")) # 0
print(get_count("")) # 0
